### PR TITLE
Add check for Riak replication (riak-repl) metrics

### PR DESCRIPTION
**Pull request by abtreece**
### What does this PR do?

Adds a custom check for Riak replication (riak-repl) starting with a subset of the available metrics

### Motivation

Replication has been included in the open source version of Riak. The node and cluster metrics are currently available in core but there is no visibility for riak-repl. 

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by abtreece** [comment]
I'm unsure if the checkboxes in the PR detail are for me or the reviewer(s) so I've left them unchecked.

**Comment by phrawzty** [comment]
👋 Hello! Thanks for the PR! One quick question before we go any further (and please forgive me if it's obvious - I'm not a Riak expert): why should this be a separate Integration? In other words, would it make sense to integrate this code / functionality into the core Riak Integration?

**Comment by abtreece** [comment]
Hey @phrawzty... the riak check in integration-core pulls stats from .../stats whereas this integration pulls from ../riak-repl/stats. The stats are provided by different Erlang "actors" and thus operate a bit independently. I suppose they *could* be combined, but I believe keeping them separate follows with the Erlang model and will allow for independent feature/fix contributions based on the advancements of those separate codebases.

**Comment by ofek** [comment]
@abtreece Hello there! Can you please fix the 2 validations in CI? Here are the valid unit names currently: https://github.com/DataDog/integrations-core/blob/a676e7b2f46e3ceb48c1d00d0abad7aa16d9a922/datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/metadata.py#L48-L184

File: riak_repl/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Events
+
+riak_repl does not currently include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog Support][6].
+
+[1]: **LINK_TO_INTEGERATION_SITE**
```
Comment (ruthnaebeck) [inline riak_repl/README.md]:
```suggestion
[1]: https://docs.datadoghq.com/integrations/riak_repl/
```

File: riak_repl/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+riak_repl does not currently include any service checks.
+
+### Events
+
+riak_repl does not currently include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog Support][6].
```
Comment (ruthnaebeck) [inline riak_repl/README.md]:
```suggestion
Need help? Contact [Datadog support][6].
```

File: riak_repl/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+2. [Restart the Agent][4]
+
+### Validation
+
+[Run the Agent's `status` subcommand][5] and look for `riak_repl` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
```
Comment (ruthnaebeck) [inline riak_repl/README.md]: The metrics below are pulled from `riak_repl/metadata.csv` when the documentation site builds. These should be deleted and replaced with the line:
See [metadata.csv][7] for a list of metrics provided by this integration.

Add this link to the bottom:
[7]: https://github.com/DataDog/integrations-core/blob/master/riak_repl/metadata.csv

Comment (abtreece) [reply riak_repl/README.md]: I believe I implemented your changes correctly... 🤞

**Comment by phrawzty** [comment]
Hi @abtreece! In order to thank you for both your contribution and your patience, I'd be happy to send you a t-shirt and some stickers. 😄 If you're interested, please shoot me an email with your postal mail details: `phrawzty at datadoghq dot com`. Thanks again!

**Pull request merged**